### PR TITLE
ROX-26590: nvdcvss migration

### PR DESCRIPTION
**Pull request by dashrews78**
### Description

Initialize the field to 0.

### User-facing documentation

- [x] CHANGELOG is updated **OR** update is not needed
- [x] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing and quality

- [x] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [ ] CI results are inspected

#### Automated testing

- [x] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests

#### How I validated my change

Added unit test.  Also started with 4.5.2 upgraded to this PR, verified the data was set to 0.  Performed a rollback to 4.5.2 and verified ACS still functioned.

### CODE REVIEW

File: migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package m207tom208
+
+import (
+	newSchema "github.com/stackrox/rox/migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+)
+
+// If so increment the `MinimumSupportedDBVersionSeqNum` to the `CurrentDBVersionSeqNum` of the release immediately
```
Comment (charmik-redhat) [inline migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go:9]: Can this comment block be removed or does the check-generated test fail if removed?

File: migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+// as 4.1 will no longer be supported. The migration process will inform the user of an error when trying to migrate
+// to a software version that can no longer be supported by the database.
+
+func migrate(database *types.Databases) error {
+	// Use databases.DBCtx to take advantage of the transaction wrapping present in the migration initiator
+	pgutils.CreateTableFromModel(database.DBCtx, database.GormDB, newSchema.CreateTableImageCvesStmt)
+
+	db := database.PostgresDB
+
+	updateStmt := `update image_cves set nvdcvss = 0 where nvdcvss is NULL`
```
Comment (charmik-redhat) [inline migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go:25]: Does serialized need an update or does it remain the same with 0 values?

Comment (dashrews78) [reply migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go:25]: The end result will be the 0 value in serialized object whether it exists or not.  I think that is a feature of protos.  I will give it some more thought between now and Monday on if we need to simply verify that or not.

File: migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go (hunk 1 above)
Comment (dashrews78) [reply migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go]: removed

File: migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go (hunk 2 above)
Comment (dashrews78) [reply migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go:15]: I did some crude testing, there is no value in putting that in the serialized object, IMO.

**Review by charmik-redhat — APPROVED** [review]

**Pull request merged**